app/views/ems_container/_heatmap.html.haml

```diff
@@ -0,0 +1,12 @@
+.heatmap{"ng-controller" => "heatmapContainerController as vm"}
+  .card-pf.card-pf-heatmap
+    .card-pf-heading
+      %h2.card-pf-title
+        {{vm.title}}
+    .card-pf-body
+      .spinner.spinner-lg.loading{"ng-if" => "!vm.loadingDone"}
+      .row
+        .col-xs-12.col-sm-12.col-md-12.col-lg-6.example-heatmap-container{"ng-repeat" => "(key, data) in vm.data"}
+          %pf-heatmap{'ng-attr-id' => "{{data[0].id}}", "chart-title" => "key", "data" => "data", "chart-data-available" => "vm.dataAvailable", "show-legend" => "vm.showLegends", "legend-labels" => "vm.legendLabels", "max-block-size" => "20", "block-padding" => "5", "heatmap-color-pattern" => "vm.heatmapColorPattern", "thresholds" => "vm.thresholds", "click-action" => "vm.clickAction", "loading-done" => true}
```

Comment:
> "chart-data-available" => "vm.dataAvailable"

I think that's the problem - dataAvailable is true before the loading is done .. looks like you may want to use `vm.dataAvailable && vm.loadingDone` here?